builder/anka/step_create_vm.go

```diff
@@ -233,3 +246,32 @@ func randSeq(n int) string {
 	}
 	return string(b)
 }
+
+func nameFromInstallerApp(path string, plb string) (string, error) {
+	info, err := os.Stat(path)
+	if os.IsNotExist(err) {
+		return "", err
+	}
+	if info.IsDir() {
+		return "", err
+	}
+
+	_, err = os.Stat(plb)
+	if os.IsNotExist(err) {
```

Comment:
If in case the `err != nil` but is not a `IsNotExist` err, does that mean you're not validating some errors?